packages/fiori/src/ShellBar.ts

```diff
@@ -436,14 +395,20 @@ class ShellBar extends UI5Element {
 	@property({ type: Boolean, noAttribute: true })
 	_fullWidthSearch!: boolean;
 
-	@property({ type: Boolean, noAttribute: true })
-	_coPilotPressed!: boolean;
-
 	@property({ type: Boolean, noAttribute: true })
 	_isXXLBreakpoint!: boolean;
 
 	/**
-	 * Defines the `ui5-shellbar` aditional items.
+	 * Defines the `ui5-shellbar` assistant.
+	 *
+	 * @since 2.0.0
+	 * @public
+	 */
+	@slot()
+	assistant!: Array<HTMLElement>;
```

Comment:
Do we support anything else than button? Can we describe the slot with IButton interface?